packages/react/src/analytics/integrations/AnalyticsApi/types/types.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  type EventData,
+  type EventTypes,
+  type IntegrationOptions,
+  type LoadIntegrationEventData,
+  type PageTypes,
+  type SetUserEventData,
+  type TrackTypesValues,
+} from '@farfetch/blackout-analytics';
+
+export interface AnalyticsApiIntegrationOptions extends IntegrationOptions {
+  conversionsAPI: {
+    eventsList: Array<`${EventTypes}` | `${PageTypes}`>;
```

Comment:
```suggestion
    eventList: Array<string>; // The user could define new event/pages types that are not in `EventTypes` or `PageTypes`
```

Reply:
ConversionsAPI only deal with some specific events, so I think it's pointless open this for any string considering eventTypes and pageTypes has all possibilities listed.